MAINTAINERS.md

```diff
@@ -24,25 +24,29 @@
 | Sayali Gaikawad | [gaiksaya](https://github.com/gaiksaya) | Amazon |
 | Abhinav Gupta | [abhinavGupta16](https://github.com/abhinavGupta16) | Amazon |
 | Zelin Hao | [zelinh](https://github.com/zelinh) | Amazon |
-| Elaine | [zylzulu](https://github.com/zylzulu) | Amazon |
+| Yilin Zhang | [zylzulu](https://github.com/zylzulu) | Amazon |
 
 [This document](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md) explains what maintainers do in this repo, and how they should be doing it. If you're interested in contributing, see [CONTRIBUTING](CONTRIBUTING.md).
 
 ## Zenhub Process Workflow
```

Comment:
I think @peternied was saying change it to `ZenHub` with both capital `Z` and `H`. You have some names as `ZenHub` and some are `Zenhub`. We should have them consistent as `ZenHub` according to their official name.